scripts/log4j2.solr.xml

```diff
@@ -0,0 +1,83 @@
+<Configuration>
+    <Appenders>
+
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout>
+                <Pattern>
+                    %maxLen{%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p (%t) [%X{collection} %X{shard} %X{replica} %X{core}] %c{1.} %m%notEmpty{ =>%ex{short}}}{10240}%n
+                </Pattern>
+            </PatternLayout>
+        </Console>
+
+        <RollingRandomAccessFile
+                name="MainLogFile"
+                fileName="${sys:solr.log.dir}/solr.log"
+                filePattern="${sys:solr.log.dir}/solr.log.%i" >
+            <PatternLayout>
+                <Pattern>
+                    %maxLen{%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p (%t) [%X{collection} %X{shard} %X{replica} %X{core}] %c{1.} %m%notEmpty{ =>%ex{short}}}{10240}%n
+                </Pattern>
+            </PatternLayout>
+            <Policies>
+                <OnStartupTriggeringPolicy />
+                <SizeBasedTriggeringPolicy size="32 MB"/>
+            </Policies>
+            <DefaultRolloverStrategy max="10"/>
+        </RollingRandomAccessFile>
+
+        <RollingRandomAccessFile
+                name="SlowLogFile"
+                fileName="${sys:solr.log.dir}/solr_slow_requests.log"
+                filePattern="${sys:solr.log.dir}/solr_slow_requests.log.%i" >
+            <PatternLayout>
+                <Pattern>
+                    %maxLen{%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p (%t) [%X{collection} %X{shard} %X{replica} %X{core}] %c{1.} %m%notEmpty{ =>%ex{short}}}{10240}%n
+                </Pattern>
+            </PatternLayout>
+            <Policies>
+                <OnStartupTriggeringPolicy />
+                <SizeBasedTriggeringPolicy size="32 MB"/>
+            </Policies>
+            <DefaultRolloverStrategy max="10"/>
+        </RollingRandomAccessFile>
+
+    </Appenders>
+    <Loggers>
+        <!-- Use <AsyncLogger/<AsyncRoot and <Logger/<Root for asynchronous logging or synchonous logging respectively -->
+        <AsyncLogger name="org.apache.hadoop" level="warn"/>
+        <AsyncLogger name="org.apache.solr.update.LoggingInfoStream" level="off"/>
+        <AsyncLogger name="org.apache.zookeeper" level="warn"/>
+        <!-- HttpSolrCall adds markers denoting the handler class to allow fine grained control, metrics are
+             very noisy so by default the metrics handler is turned off to see metrics logging set DENY to ACCEPT -->
+        <AsyncLogger name="org.apache.solr.servlet.HttpSolrCall" level="info">
+            <MarkerFilter marker="org.apache.solr.handler.admin.MetricsHandler" onMatch="DENY" onMismatch="ACCEPT"/>
+        </AsyncLogger>
+        <AsyncLogger name="org.apache.solr.core.SolrCore.SlowRequest" level="info" additivity="false">
+            <AppenderRef ref="SlowLogFile"/>
+        </AsyncLogger>
+
+        <AsyncRoot level="info">
+            <AppenderRef ref="MainLogFile"/>
+            <AppenderRef ref="STDOUT" level="WARN"/>
```

Comment:
Solr log setup file is the original one with this small change (level="WARN")

Reply:
add comment into the file